src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -84,6 +84,12 @@ internal static class FeatureOnOffOptions
                 defaultValue: true,
                 new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ShowInheritanceMargin"));
 
+        public static readonly PerLanguageOption2<bool> InheritanceMarginCombinedWithIndicatorMargin =
+            new(nameof(FeatureOnOffOptions),
+                nameof(InheritanceMarginCombinedWithIndicatorMargin),
+                defaultValue: false,
```

Comment:
Maybe we should change the default value to true to maintain the old behavior by default. And the unhappy customer would just manually turn this on.

Reply:
yes.  please do this.  yes.  yes. yes.

Reply:
OK, I will update this later.